docs/install-crossplane.md

```diff
@@ -269,6 +269,7 @@ The following tables lists the configurable parameters of the Crossplane chart a
 | `clusterStacks.rook.version`     | Rook stack version to deploy                                    | `<latest released version>`   
 | `personas.deploy`                | Install roles and bindings for Crossplane user personas         | `true`     
 | `templateStacks.enabled`         | Enable experimental template stacks support                     | `false`     
+| `templateStacks.controllerImage` | Template Stack controller image                                 | `crossplane/templating-controller:0.1.0`
```

Comment:
I didn't realize this table lived in more than one place o:

Reply:
See #1109 and #1239